sentry-opentelemetry/sentry-opentelemetry-agentless-spring/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    `java-library`
+}
+
+configure<JavaPluginExtension> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+dependencies {
+    api(projects.sentry)
+    api(projects.sentryOpentelemetry.sentryOpentelemetryBootstrap)
+    implementation(projects.sentryOpentelemetry.sentryOpentelemetryAgentcustomization)
+    api(Config.Libs.OpenTelemetry.otelSdk)
+    api(Config.Libs.OpenTelemetry.otelSemconv)
+    api(Config.Libs.OpenTelemetry.otelSemconvIncubating)
+    api(Config.Libs.OpenTelemetry.otelExtensionAutoconfigure)
```

Comment:
Are we not depending on the `sentry-opentelemetry-agentless` module because of the `AgentlessMarker`?

Reply:
Yeah, I thought, it might cause more issues than save work to reuse the other `agentless` module here since there's very little in both currently.